verde/base/base_classes.py

```diff
@@ -147,7 +149,9 @@ def filter(self, coordinates, data, weights=None):
         ----------
         coordinates : tuple of arrays
             Arrays with the coordinates of each data point. Should be in the
-            following order: (easting, northing, vertical, ...).
+            following order: (easting, northing, vertical, ...)
+            or (longitude, latitude, height, ...)
+            or (longitude, latitude, radius, ...).
```

Comment:
This might get a bit confusing. How about this:

```suggestion
            following order: (easting, northing, vertical, ...). 
            For the specific definition of coordinate systems and what these 
            names mean, see the class docstring. 
```
Then the class can define that the system is spherical and these mean longitude, latitude, radius, for example.